src/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceEventSourceBridgeTests.cs

```diff
@@ -464,8 +534,11 @@ public TestDiagnosticSourceEventListener()
 
         public int EventCount;
         public DiagnosticSourceEvent LastEvent;
-        public DiagnosticSourceEvent SecondLast;
+#if DEBUG 
+        // Here just for debugging.  Lets you see the last 3 events that were sent.  
+        public DiagnosticSourceEvent SecondLast;  
```

Comment:
The code in the DEBUG ifdefs was added solely so I could see some information in the debugger's watch window.  I wanted a way of indicating that this code as not important to the functionality or design (you could ignore it when trying to understand the code).     It is a minor point one way or the other.